lib/internal/test_runner/utils.js

```diff
@@ -270,6 +274,14 @@ function parseCommandLine() {
   if (coverage) {
     coverageExcludeGlobs = getOptionValue('--test-coverage-exclude');
     coverageIncludeGlobs = getOptionValue('--test-coverage-include');
+
+    minimumBranchCoveragePercent = getOptionValue('--test-coverage-min-branch');
+    minimumLineCoveragePercent = getOptionValue('--test-coverage-min-line');
+    minimumFunctionCoveragePercent = getOptionValue('--test-coverage-min-function');
+
+    validateNumber(minimumBranchCoveragePercent, '--test-coverage-min-branch', 0, 100);
```

Comment:
We should probably use `validateInteger()` here since floating point values are not accepted based on the flags added to the CLI.